### ISSUE TITLE
Form & CRUD

### ISSUE THREAD

**Issue by cjk2** [ridwanskaterock/cicool#313]
I would like to edit a table comfortably with my colleagues in the backend. From my point of view, this also includes an appealing form. Can I connect forms directly to the databases (CRUD)? Sorry for the stupid question. I am rather a beginner.

**Comment by ridwanskaterock** [comment]
yes you can for that you can generate the form and then change the table in the form model to the CRUD table

![image](https://user-images.githubusercontent.com/9270445/64015551-f7652780-cb4e-11e9-9526-7e237436252d.png)


change on this
